src/nl/hannahsten/texifyidea/action/wizard/table/LatexTableWizardAction.kt

```diff
@@ -48,18 +47,26 @@ class LatexTableWizardAction : AnAction() {
                     project,
                     "Insert table",
```

Comment:
I'm getting a warning 'It should have title capitalization'. Don't know if you agree, we never did this until this inspection came up